README.md

```diff
@@ -34,3 +34,9 @@ If you are interested in building the extensions locally, please take a look at
 You can find more information about developing Salesforce Extensions for VS Code in the [docs](docs) folder. If the docs don’t cover what you are looking for, please feel free to open an issue.
 
 For information about using the extensions, consult the README.md file for each package.
+
+### Be An Efficient Salesforce Developer with VS Code
+
+Dreamforce 2018 session on how to use VS Code and the SalesforceDX Extensions for VS Code
+
+[![Be An Efficient Salesforce Developer with VS Code](imgs/DF18_VSCode_Session_thumbnail.jpg)](https://www.youtube.com/watch?v=hw9LBvjo4PQ)
```

Comment:
There's no reliable way of embedding a youtube video on markdown so this is the next best thing. I've added an image to the repo instead of using the thumbnails provided by the YouTube api since this looks more like a clickable video than the thumbnails provided by YouTube.

Reply:
I think this should go _before_ the `Getting Started` section.

It feels a bit weird to have general information in the code repo after the more technical and specific `Getting Started` section.